### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by foovungle**
Connector -> CassandraConnector in the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ performance improvement over query performance. Async iterators needed a lot of
  dumping C* data into HDFS or whatever backup system. A big 60 - 70% gain should be expected.
 
 Phantom connectors now require an ```implicit com.websudos.phantom.connectors.KeySpace``` to be defined. Instead of using a plain string, you just have to
-use ```KeySpace.apply``` or simply: ```trait MyConnector extends Connector { implicit val keySpace = KeySpace("your_def") } ```. This change allows us to
+use ```KeySpace.apply``` or simply: ```trait MyConnector extends CassandraConnector { implicit val keySpace = KeySpace("your_def") } ```. This change allows us to
```
Comment (alexflav23) [inline README.md]: Thank you for correcting that. I think we may better go with `SimpleCassandraConnector`, as that will only require defining an implicit keyspace into scope and it's already available in the default imports.

Comment (foovungle) [inline README.md]: That's a good point. Disregard my pull request, but please correct the README. Took me a little bit of hunting to find the right class to extend.

Comment (alexflav23) [inline README.md]: @foovungle Fair game, we are working on a brand new wiki for Phantom which will do a far better job at introducing you to phantom + several tutorials.

**Comment by alexflav23** [comment]
Hi @foovungle , we've moved all documentation to Github Wiki and fixed all broken links in the process. I will now close this PR as it's out of date.

**Comment by foovungle** [comment]
Thanks

**Pull request closed without merging**